web/modules/custom/demo_content/content/rdf_entity/solution/core-person-vocab.json

```diff
@@ -56,7 +56,7 @@
     },
     "id": [
         {
-            "value": "http:\/\/www.w3.org\/ns\/person#"
```

Comment:
Good question. That was a temporary solution for the issue we were facing with the id. I _was_ planning to revert. Good that you caught it.